src/olympia/files/utils.py

```diff
@@ -968,8 +968,9 @@ def resolve_i18n_message(message, messages, locale, default_locale=None):
     if match is None:
         return message
 
+    locale = find_language(locale)
+    default_locale = find_language(locale)
```

Comment:
Shouldn't it be `find_language(default_locale)` ? Also, if `default_locale` is `None`, that'd be bad - I assume it's checked by the caller ?